examples/remove_CR_hits.py

```diff
@@ -0,0 +1,148 @@
+"""
+========================
+Removing Cosmic Ray Hits
+========================
+
+This example illustrates how to remove cosmic ray hits from a LASCO C2 image (using ``FITS`` and ``jp2``)
+using ``astroscrappy.detect_cosmics <https://astroscrappy.readthedocs.io/en/latest/api/astroscrappy.detect_cosmics.html>``__. Astroscrappy is a separate Python package and can be
+installed separately using ``pip`` or ``conda``.
+"""
+# sphinx_gallery_thumbnail_number = 2
+
+import matplotlib.pyplot as plt
+
+from sunpy.map import Map
+
+import astroscrappy
+
+###############################################################################
+# First, we will work with the FITS files.
+# These imports are necessary to download the FITS file based on the instrument.
+# For more details on how to download a particular LASCO FITS file as a map see
+# Sunpy's example ``Downloading and plotting LASCO C3 data <https://docs.sunpy.org/en/stable/generated/gallery/acquiring_data/skip_downloading_lascoC3.html#sphx-glr-generated-gallery-acquiring-data-skip-downloading-lascoc3-py>``_.
+# To make this example work you need to have ``Sunpy`` with all the `net` dependencies also installed.
+from sunpy.net import Fido, attrs as a
+from sunpy.io.file_tools import read_file
+
+###############################################################################
+# We will also work with ``jp2`` images. So to download these files we will use
+# Sunpy's Helioviewer.org client. For more information about the helioviewer client,
+# see sunpy example ``Querying Helioviewer.org with SunPy <https://docs.sunpy.org/en/stable/guide/acquiring_data/helioviewer.html>``_.
+from sunpy.net.helioviewer import HelioviewerClient
+
+###############################################################################
+# In order to download the required FITS file, we use
+# `Fido <sunpy.net.fido_factory.UnifiedDownloaderFactory>`, a downloader client.
```

Comment:
Will this reference be found here? as it is not sunpy

Reply:
Assuming I set it up correctly, it should.